src/components/snapshot/overview/SnapshotOverviewPanel.tsx

```diff
@@ -204,7 +215,16 @@ function SnapshotOverviewPanel(props: SnapshotOverviewPanelProps) {
           </Grid>
         )}
       </TabPanel>
-      <TabPanel value={value} index={2}>
+      {canViewJournalEntries && (
+        <TabPanel value={value} index={2}>
+          <Grid container spacing={2}>
+            <Grid item xs={12}>
+              <JournalEntriesView id={snapshot.id} resourceType="DATASNAPSHOT" />
```

Comment:
We should probably have a constant for DATASNAPSHOT (and DATASET) in constants/index.ts

Reply:
Oh, I found one for this in the generated tdr values! :) Updated this code with it!